tests_v2/test_focus_resize.py

```
@@ -0,0 +1,231 @@
+
+        print("PASS: focus-resize correctly adjusts pane geometry on focus change")
+        return 0
+
+    finally:
+        _disable_focus_resize()
+        if workspace_id:
+            try:
+                with cmux(socket_path) as cleanup:
+                    cleanup.close_workspace(workspace_id)
+            except Exception:
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)